test/unit/geojson.js

```diff
@@ -17,12 +17,12 @@ function parse(geojson) {
 describe('GeoJsonParser', function () {
     it('should set all z coordinates to 1', () =>
```

Comment:
Unless I'm missing something, shouldn't this description be `'should set all z coordinates to 0'` ?